alpine/resin

```diff
@@ -0,0 +1,66 @@
+#!/sbin/openrc-run
```

Comment:
@petrosagg, I created our init script based on `start-top-daemon` manpages, it's easy to find them. But we're talking about simpler templates like https://aur.archlinux.org/cgit/aur.git/tree/ModemManager?h=modemmanager-openrc or https://wiki.alpinelinux.org/wiki/Writing_Init_Scripts. it's `the start/stop templates, I can't find manpages for them and clueless about what are the available `special predefined variables`.